airc/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
-name = "air-script"
-version = "0.0.1"
+name = "airc"
```

Comment:
I would keep the name `air-script` as the name of the package as this is what will go on crates.io. But we can still change the name of the executable to be `airc`.